source/release-notes/1.8.txt

```diff
@@ -21,9 +21,91 @@ Release Notes for mongosync 1.8
+verifier that performs a series of checks against the source and 
+destination clusters to verify a successful sync.
+
+For more information, see :ref:`c2c-embedded-verifier`.
+
+Other Notes
+~~~~~~~~~~~
+
+Updates to ``mongosync`` ``/progress`` API endpoint:
+
+- The ``/progress`` endpoint now returns a top-level boolean 
+  ``success`` field.
+
+- The ``/progress`` endpoint now returns the integer 
+  ``totalEventsApplied`` field, which approximates the number of 
+  change events applied to a destination cluster by an instance of 
+  ``mongosync``.
+
+- The ``/progress`` endpoint now reports per-partition progress 
+  tracking with the following changes:
+
+  - Calling ``/progress`` before ``mongosync`` reaches the collection 
+    copy phase will return 0 for ``estimatedCopiedBytes`` and 1 for 
+    ``estimatedTotalBytes``.
+
+  - ``estimatedTotalBytes`` may change throughout the collection copy 
+    phase if documents are inserted or deleted on the source cluster.
+
+  - ``estimatedCopiedBytes`` will never be greater than 
+    ``estimatedTotalBytes``. Progress will reach 100% at the end of 
+    the collection copy phase 
+    (``estimatedCopiedBytes`` = ``estimatedTotalBytes``).
+
+  - When performing a live upgrade from an earlier version to 1.8.0 or 
+    higher, the collection copy data will start over from 
+    ``[0 bytes copied / 1 bytes total]``. After a live upgrade, 
+    ``/progress`` only reports the progress of data copied after the 
+    upgrade completed.
+
+Optimizations:
+
+- Significantly reduced the time needed to create partitions and added logging 
+  that tracks the partition creation process.
+
+Other Changes:
+
+- ``mongosync`` now performs a round robin assignment of the primary 
+  shard for each database if the destination is a sharded cluster.
+
+- ``mongosync`` now errors if it cannot create an index after 10 
+  attempts. Previously, the system would attempt to create an index 
+  indefinitely.
+
+- ``mongosync`` now sends hostnames as telemetry.
+
+- Improved performance of the initialization process by eliminating 
+  the creation of unnecessary dummy indexes. ``mongosync`` now creates 
+  dummy indexes for sharded collections only if the destination 
+  cluster does not have an index to support the shard key.
+
+Issues Fixed:
+
+- Fixed a bug that caused ``mongosync`` to fall off the source 
+  cluster's oplog if no real writes occurred on the source 
+  cluster for an extended period of time.
+
+- Fixed a bug that caused ``mongosync`` to write certain logs to a 
+  location outside of the specified log directory.
+
+- Fixed a bug that caused the ``mongosync`` ``/progress`` endpoint to 
+  potentially return incorrect ``totalBytes`` for sharded clusters.
+
+- Fixed a bug that could decrease the ``size`` of capped collections 
+  by up to 256 bytes. This bug affected capped collection resizing 
+  when migrating from a source with a version <7.0 to a destination 
```

Comment:
(wording)
I'd recommend using less than and greater than when possible.

Reply:
Done